utilities/metadata/project.schema.json

```diff
@@ -158,19 +200,152 @@
           "items": {
             "$ref": "#/$defs/hardware_descriptor"
           }
+        },
+        "external": {
+          "type": "array",
+          "items": {
+            "oneOf": [
+              {
+                "type": "string"
+              },
+              {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string"
+                  },
+                  "description": {
+                    "type": "string"
+                  }
+                },
+                "required": ["name", "description"]
+              }
+            ]
+          }
         }
       }
     },
     "run_command": {
       "type": "object",
       "properties": {
         "command": {
-          "type": "string"
+          "type": "string",
+          "description": "Command to execute. Supports parameter substitution using ${param_name}, ${env:VAR_NAME}, and ${holohub:placeholder}"
         },
         "workdir": {
+          "type": "string",
+          "description": "Working directory for command execution"
+        },
+        "run_args": {
```

Comment:
Can we merge `command` and `run_args` when under `modes` (keep them separate for backward compatibility under the top level `run` for now) ?

Reply:
cc @tbirdso @jjomier